### PR TITLE
Teams App Localization using tab and bot

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
**Description**:
Added a Bot and Tab for teams App localization.

**Language used: C#**

### CODE REVIEW

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
Could you please move this under **samples/app-localization/csharp/**

**Comment by Trinetra-MSFT** [comment]
> Could you please move this under **samples/app-localization/csharp/**

Done moved this to new folder

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
Could you please remove unused manifest files? 
oneMOreTest.zip
testAgain.zip
localization-manifest.zip

Keep only single file.

File: samples/app-localization/csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/msft-teams/tools.git
```
Comment (Wajeed-msft) [inline samples/app-localization/csharp/README.md:13]: Please use the correct repo path.

File: samples/app-localization/csharp/Localization/Bots/MessageExtension.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {            
+            turnContext.Activity.RemoveRecipientMention();
+            var text = turnContext.Activity.Text.Trim();        
+
+            var replyText = $"You said:"+ _localizer[text];
+            await turnContext.SendActivityAsync(MessageFactory.Text(replyText, replyText), cancellationToken);            
+        }
+
+        protected override Task<MessagingExtensionResponse> OnTeamsMessagingExtensionQueryAsync(ITurnContext<IInvokeActivity> turnContext, MessagingExtensionQuery query, CancellationToken cancellationToken)
```
Comment (Wajeed-msft) [inline samples/app-localization/csharp/Localization/Bots/MessageExtension.cs:33]: Could you please remove unused code?

File: samples/app-localization/csharp/Localization/Bots/MessageExtension.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+using Newtonsoft.Json.Linq;
+using System.Linq;
+using System;
+using System.Collections.Generic;
+using Bogus;
+using Microsoft.Extensions.Localization;
+
+namespace Localization
+{
+    public class MessageExtension : TeamsActivityHandler
```
Comment (Wajeed-msft) [inline samples/app-localization/csharp/Localization/Bots/MessageExtension.cs:15]: Could you please rename this to more appropriate class name?

**Review by Wajeed-msft — COMMENTED** [review]
Please add link to this sample from main section. Put this under [scenario-specific-samples](https://github.com/BobGerman/Microsoft-Teams-Samples#scenario-specific-samples) section.

File: samples/app-localization/csharp/Localization/Bots/MessageExtension.cs (hunk 3 above)
Comment (Trinetra-MSFT) [reply samples/app-localization/csharp/Localization/Bots/MessageExtension.cs]: Renamed to LocalizerBot

File: samples/app-localization/csharp/Localization/Bots/MessageExtension.cs (hunk 2 above)
Comment (Trinetra-MSFT) [reply samples/app-localization/csharp/Localization/Bots/MessageExtension.cs]: Removed unused code

File: samples/app-localization/csharp/README.md (hunk 1 above)
Comment (Trinetra-MSFT) [reply samples/app-localization/csharp/README.md]: Corrected repo path

Comment (Trinetra-MSFT) [reply samples/app-localization/csharp/README.md]: Updated Main ReadMe file

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**